### DISCUSSION TITLE
Saleor-Platform Installation Didn't work

### DISCUSSION THREAD

**Discussion by Swanhtet-grc** [saleor/saleor#5955] [Q&A]
**The problem** 

Saleor Documentation said to use 

`git clone https://github.com/mirumee/saleor-platform.git --recursive --jobs 3`

to install the saleor-platform. But it didn't pull sub-modules.

I got nothing in saleor, saleor-storefront and saleor-dashboard folder.

**How I solved it**

I did

`git clone https://github.com/mirumee/saleor-platform.git `
`cd saleor-platform`
`rmdir saleor saleor- storefront saleor-dashboard`
`git clone https://github.com/mirumee/saleor.git`
`git clone https://github.com/mirumee/saleor-dashboard.git`
`git clone https://github.com/mirumee/saleor-storefront.git`
`docker-compose build`

I don't know it is the right solution. I'm new here, so please point me out.

**Comment by robertology** [comment]
The first time I tried that, I got some github access errors. I had to `ssh-add` my key first. I've done the full clone, etc process a few times this week and have had no issues. I would try again and look for any error messages.

**Comment by Swanhtet-grc** [comment]
Okay. Thank you.

**Comment by neerajgupta2407** [comment]
@Swanhtet-grc Hey,  i hope this might help. https://www.youtube.com/watch?v=AKHUyKu59hI&t=343s

**Comment by Swanhtet-grc** [comment]
I already got the solution. I opened this issue because I just would to check my solution is the right solution or not.
And I would like to check the official docs of Saleor is right.
